### PR TITLE
[FLINK-12944][docs]Translate Streaming File Sink page into Chinese

### PR DESCRIPTION
**Pull request by aloyszhang**
## What is the purpose of the change

Translate Streaming File Sink page into Chinese

## Brief change log

Translate Streaming File Sink page into Chinese

## Verifying this change

This change is a trivial rework / code cleanup without any test coverage.

## Does this pull request potentially affect one of the following parts:

  - Dependencies (does it add or upgrade a dependency): (no)
  - The public API, i.e., is any changed class annotated with `@Public(Evolving)`: ( no)
  - The serializers: (no)
  - The runtime per-record code paths (performance sensitive): (no )
  - Anything that affects deployment or recovery: JobManager (and its components), Checkpointing, Yarn/Mesos, ZooKeeper: (no)
  - The S3 file system connector: ( no)

## Documentation

  - Does this pull request introduce a new feature? ( no)

### CODE REVIEW

**Comment by aloyszhang** [comment]
@wuchong

File: docs/dev/connectors/streamfile_sink.zh.md [hunk 1]
```diff
@@ -23,30 +23,23 @@ specific language governing permissions and limitations
 under the License.
 -->
 
-This connector provides a Sink that writes partitioned files to filesystems
-supported by the [Flink `FileSystem` abstraction]({{ site.baseurl}}/ops/filesystems/index.html).
+这个连接器支持向 [Flink `FileSystem` abstraction]({{ site.baseurl}}/zh/ops/filesystems/index.html)支持
```
Comment (wuchong) [inline docs/dev/connectors/streamfile_sink.zh.md]:
```suggestion
这个连接器提供了一个 Sink 来将分区文件写入到支持 [Flink `FileSystem`]({{ site.baseurl}}/zh/ops/filesystems/index.html) 接口的文件系统中。
```

File: docs/dev/connectors/streamfile_sink.zh.md [hunk 2]
```diff
@@ -23,30 +23,23 @@ specific language governing permissions and limitations
...
+的文件系统写入分区文件。
```
Comment (wuchong) [inline docs/dev/connectors/streamfile_sink.zh.md]:
```suggestion
```

File: docs/dev/connectors/streamfile_sink.zh.md [hunk 3]
```diff
@@ -23,30 +23,23 @@ specific language governing permissions and limitations
...
-individual files that each contain a part of the infinite output stream.
+由于在流处理中输入可能是无限的，所以流处理的文件 sink 会将数据写入到桶中。如何分桶是可以配置的，一种有效的默认
+策略是基于时间的分桶，这种策略每个小时写入一个新的桶，这些桶各包含了无限输出流的一部分数据。
 
-Within a bucket, we further split the output into smaller part files based on a
-rolling policy. This is useful to prevent individual bucket files from getting
-too big. This is also configurable but the default policy rolls files based on
-file size and a timeout, *i.e* if no new data was written to a part file.
+在一个桶内部，会进一步将输出基于滚动策略切分成更小的文件。这有助于防止桶文件变得过大。滚动策略也是可以配置的，默认
+策略会根据文件大小和超时时间来滚动文件，超时时间是指没有新数据写入部分文件的时间。
```
Comment (wuchong) [inline docs/dev/connectors/streamfile_sink.zh.md]:
```suggestion
策略会根据文件大小和超时时间来滚动文件，超时时间是指没有新数据写入部分文件（part file）的时间。
```

这里我觉得 part file 是 Stream File Sink 的一个专用术语，最好能显示原文，更容易让用户理解。

File: docs/dev/connectors/streamfile_sink.zh.md [hunk 4]
```diff
@@ -84,55 +77,41 @@ input.addSink(sink)
...
-#### Using Bulk-encoded Output Formats
+#### 使用批量编码输出格式
 
-In the above example we used an `Encoder` that can encode or serialize each
-record individually. The streaming file sink also supports bulk-encoded output
-formats such as [Apache Parquet](http://parquet.apache.org). To use these,
-instead of `StreamingFileSink.forRowFormat()` you would use
-`StreamingFileSink.forBulkFormat()` and specify a `BulkWriter.Factory`.
+上面的示例使用 `Encoder` 分别序列化每一个记录。除此之外，流式文件 sink 还支持批量编码的输出
+格式，比如 Apache Parquet](http://parquet.apache.org)。使用这种编码格式需要用 `StreamingFileSink.forBulkFormat()`
```
Comment (wuchong) [inline docs/dev/connectors/streamfile_sink.zh.md]:
```suggestion
格式，比如 [Apache Parquet](http://parquet.apache.org)。使用这种编码格式需要用 `StreamingFileSink.forBulkFormat()`
```

File: docs/dev/connectors/streamfile_sink.zh.md [hunk 5]
```diff
@@ -84,55 +77,41 @@ input.addSink(sink)
...
 
 [ParquetAvroWriters]({{ site.javadocs_baseurl }}/api/java/org/apache/flink/formats/parquet/avro/ParquetAvroWriters.html)
-has static methods for creating a `BulkWriter.Factory` for various types.
+中包含了为各种类型创建 `BulkWriter.Factory` 的静态方法。
 
 <div class="alert alert-info">
-    <b>IMPORTANT:</b> Bulk-encoding formats can only be combined with the
-    `OnCheckpointRollingPolicy`, which rolls the in-progress part file on
-    every checkpoint.
+    <b>IMPORTANT:</b> 批量编码格式只能和 `OnCheckpointRollingPolicy` 结合使用，每次做 checkpoint 时滚动文件。
```
Comment (wuchong) [inline docs/dev/connectors/streamfile_sink.zh.md]:
```suggestion
    <b>重要:</b> 批量编码格式只能和 `OnCheckpointRollingPolicy` 结合使用，每次做 checkpoint 时滚动文件。
```

File: docs/dev/connectors/streamfile_sink.zh.md [hunk 6]
```diff
@@ -84,55 +77,41 @@ input.addSink(sink)
...
-For inactive MPUs, S3 supports a bucket lifecycle rule that the user can use to abort multipart uploads
-that don't complete within a specified number of days after being initiated. This implies that if you set this rule
-aggressively and take a savepoint with some part-files being not fully uploaded, their associated MPUs may time-out
-before the job is restarted. This will result in your job not being able to restore from that savepoint as the
-pending part-files are no longer there and Flink will fail with an exception as it tries to fetch them and fails.
+####  关于S3的重要内容
+
+<span class="label label-danger">重要提示 1</span>: 对于 S3，`StreamingFileSink`  只支持基于 [Hadoop](https://hadoop.apache.org/) 
+的文件系统实现，不支持基于 [Presto](https://prestodb.io/) 的实现。如果想使用 `StreamingFileSink` 向 S3 写入数据并且将 
+checkpoint 放在基于 Presto 的文件系统，建议在 sink 路径中明确指定 *"s3a://"* 模式，在 checkpoint 路径中明确指定 *"s3p://"*
```
Comment (wuchong) [inline docs/dev/connectors/streamfile_sink.zh.md]:
```suggestion
checkpoint 放在基于 Presto 的文件系统，建议明确指定 *"s3a://"* （为 Hadoop）作为sink的目标路径方案，并且为 checkpoint 路径明确指定 *"s3p://"* （为 Presto）
```

File: docs/dev/connectors/streamfile_sink.zh.md [hunk 7]
```diff
@@ -84,55 +77,41 @@ input.addSink(sink)
...
-that don't complete within a specified number of days after being initiated. This implies that if you set this rule
-aggressively and take a savepoint with some part-files being not fully uploaded, their associated MPUs may time-out
-before the job is restarted. This will result in your job not being able to restore from that savepoint as the
-pending part-files are no longer there and Flink will fail with an exception as it tries to fetch them and fails.
+####  关于S3的重要内容
+
+<span class="label label-danger">重要提示 1</span>: 对于 S3，`StreamingFileSink`  只支持基于 [Hadoop](https://hadoop.apache.org/) 
+的文件系统实现，不支持基于 [Presto](https://prestodb.io/) 的实现。如果想使用 `StreamingFileSink` 向 S3 写入数据并且将 
+checkpoint 放在基于 Presto 的文件系统，建议在 sink 路径中明确指定 *"s3a://"* 模式，在 checkpoint 路径中明确指定 *"s3p://"* 
+模式。如果 Sink 和 checkpoint 都使用 *"s3://"* 模式，可能会导致不可预知的行为。
```
Comment (wuchong) [inline docs/dev/connectors/streamfile_sink.zh.md]:
```suggestion
如果 Sink 和 checkpoint 都使用 *"s3://"* 路径的话，可能会导致不可预知的行为，因为双方的实现都在“监听”这个路径。
```

注意 scheme 不是模式的意思

**Comment by aloyszhang** [comment]
@wuchong 
Thanks for your suggestions. Fixed.

**Pull request closed without merging**